config/prow/config.yaml

```
@@ -226,6 +226,20 @@ branch-protection:
             users: []
             teams:
             - stage-bots
+        minikube:
+          enforce_admins: true
+          restrictions: # only allow admins to force-push
+            users: []
+            teams:
+            - stage-bots
```

As this isn't a staging repo, you can just leave this blank:
```suggestion
            teams: []
```